packages/odyssey-react-mui/src/CheckboxGroup.tsx

```diff
@@ -53,16 +49,13 @@ const CheckboxGroup = ({
   children,
   errorMessage,
   hint,
-  id: idOverride,
   isDisabled,
   isRequired = false,
   label,
 }: CheckboxGroupProps) => {
   const renderFieldComponent = useCallback(
-    ({ ariaDescribedBy, id }) => (
-      <MuiFormGroup aria-describedby={ariaDescribedBy} id={id}>
-        {children}
-      </MuiFormGroup>
+    ({ ariaDescribedBy }) => (
+      <MuiFormGroup aria-describedby={ariaDescribedBy}>{children}</MuiFormGroup>
```

Comment:
So this element doesn't take an `id`?

Reply:
There's no reason for a CheckboxGroup to take an id, since it wouldn't get applied to any of the child checkboxes — those need to take their own id. (Unlike a RadioGroup, the checkboxes within a CheckboxGroup are independent from one another.)

Reply:
Is there a reason we want to omit `id` specifically? There are some normal cases like targeting the `fieldset` with a `for` on an exterior label.